### PR TITLE
Math magician components

### PR DESCRIPTION
**Pull request by Koshao-sys**
Hello,

**In this pull request, I have implemented the following requirements;**

- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches the shared design.
- Implement the design only (JSX and CSS)
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use functional components
- Create at least 1 child component
- Once the Calculator is ready, import it and use it in your top-level component: App.js
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const Items = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+  return (
+    <div className="key-area">
+      {Items.map((Item, ind) => (
+        <button key={uuidv4()} id={`btn-${ind}`} type="submit" className="key-btn">{Item}</button>
+      ))}
+    </div>
+  );
+}
+
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [ ] Good job creating these child components, however, we need to move these components to their respective new files. Kindly do that. Generally speaking, in React each component should have its own file for better code organization. You can export and them import them into this calculator component.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Koshao-sys,

Good job so far!
✔️ Use of child components
✔️ Creation of functional components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly remove these boilerplate files.

  image

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Koshao-sys) [reply src/components/Calculator.js]: @TedLivist Thanks for the insight, though the exercise didn't seem like it required the separation of the child components in different files. I will still go ahead and adjust as recommended. Thanks

**Review by levy002 — APPROVED** [review]
## Hello @Koshao-sys 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**